bccsp/factory/opts.go

```diff
@@ -1,21 +1,20 @@
 /*
-Copyright IBM Corp. 2016 All Rights Reserved.
+Copyright IBM Corp. All Rights Reserved.
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-		 http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
+SPDX-License-Identifier: Apache-2.0
 */
 
 package factory
 
+import "github.com/hyperledger/fabric/bccsp/pkcs11"
+
+// FactoryOpts holds configuration information used to initialize factory implementations
+type FactoryOpts struct {
+	Default string             `json:"default" yaml:"Default"`
```

Comment:
I realize you're just moving this code about, but it's really bothersome that the yaml and json keys are differently cased.  I'm not sure who would be depending on this behavior, so perhaps it's safest not to change, but just have to point it out.